test/functional/wallet_balance.py

```diff
@@ -50,7 +50,9 @@ def set_test_params(self):
         self.num_nodes = 2
         self.setup_clean_chain = True
         self.extra_args = [
-            ['-limitdescendantcount=3'],  # Limit mempool descendants as a hack to have wallet txs rejected from the mempool
+            # Limit mempool descendants as a hack to have wallet txs rejected from the mempool.
+            # Set walletrejectlongchains=false so the wallet still creates the transactions.
+            ['-limitdescendantcount=3', '-walletrejectlongchains=false'],
```

Comment:
nit: pretty sure you can't use boolean values here. (Otherwise the test should fail with `=true`, which I presume it doesn't ?)

```suggestion
            # Limit mempool descendants as a hack to have wallet txs rejected from the mempool.
            # Set walletrejectlongchains=0 so the wallet still creates the transactions.
            ['-limitdescendantcount=3', '-walletrejectlongchains=0'],
```

Reply:
Fixed